### ISSUE TITLE
Remove Sentry in favor of Datadog APM & RUM

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#6071]
Now that we have Datadog RUM on the frontend, we have no reason to keep Sentry. Everything we're sending to Sentry we're already sending to datadog.

Datadog is superior because it can correlate client events with server events, nginx, rethinkdb. It can do client-side session replays, and it's 1 fewer tool that our devs need to learn. 

AC
- [ ] Explore datadog & sentry. Determine if datadog receiving all the events that we send to sentry today. If not, create an issue to get us there.
- [ ] Create an issue to replace all calls to sentry with calls to datadog & remove sentry from our app.
- [ ] Create an issue to document how to discover errors in datadog like we do in sentry today

**Comment by blimmer** [comment]
See also #5991 - it would be nice to get that merged before attacking this. It'll help reduce merge conflicts adjusting Sentry.

**Comment by blimmer** [comment]
@mattkrick, as part of the AC, should we include updates to the [governance policy 🔒  ](https://www.notion.so/parabol/Sentry-error-monitoring-2349517d5af642a88b1d414a3af18f0f) related to Sentry?

**Comment by Dschoordsch** [comment]
Stuck on #6731

**Comment by Dschoordsch** [comment]
While the full stack trace is now available, I ran into some process issue. Creating and linking a GitHub issue seems not to be possible. This is not too bad, but I did not find a way to assign someone or otherwise mark an issue as being processed. The only thing I found was setting the status from "open" to "ignored".
I contacted support about this, maybe there is a better way I could not find.

**Comment by jordanh** [comment]
Going to keep this issue open

**Comment by nickoferrall** [comment]
> I contacted support about this, maybe there is a better way I could not find.

@Dschoordsch curious if you heard back from them?

**Comment by Dschoordsch** [comment]
@nickoferrall Yes, I had heard back from them, there is no way of marking an issue as assigned, i.e. to exclude it from a search filter in the future. I feel like this is a main blocker.
I looked again about a month ago and could not find anything new.

**Comment by enriquesanchez** [comment]
Iceboxed

**Comment by Dschoordsch** [comment]
Should now be possible according to @rafaelromcar-parabol

**Comment by rafaelromcar-parabol** [comment]
Idea: during this work, make it optional to use the APM. In many environments there is no APM and it shouldn't be mandatory to use. Example: self-hosted, PubSec envs, PPMIs...

If enabled, send stuff to the APM, if disabled, just throw logs!

**Comment by Dschoordsch** [comment]
I'm checking if we get the errors we see in Sentry also in Datadog by finding the corresponding errors for the default filter

<img width="1236" alt="Image" src="https://github.com/user-attachments/assets/b0918dd1-adc0-4a9c-bd52-ac97ca8c8510" />

* close enough
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/5ccbfa68-cbd1-475e-b1d2-a5971b74d9ca" />  
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/312728ed-2e57-4b61-9e23-b523145a432e" />
* datadog grouped it differently
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/7ac5a799-2678-4729-8921-9f414b8a48ed" />
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/2c9c52df-378d-4ce2-b4f9-5e69c40a1826" />
* close enough
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/d10be7c8-7d10-4ae2-a56b-677683892ea1" />
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/96cce4a3-6e83-472a-9ba8-77300c6497ec" />
* this one includes sentry in the errors, so I expect some of these to go away if we'd switch it off
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/e64fd9cf-0b5b-44a7-a8e1-9c65319134bf" />
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/8d72911c-4198-499e-973d-1cd95d47dbd9" />
* close enough
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/55c6054c-e22a-4baf-944b-19e4729561e0" />
  <img width="1236" alt="Image" src="https://github.com/user-attachments/assets/4bc42ce2-0607-4833-93bf-1f1f175accf4" />

I don't think we would miss any errors when checking datadog exclusively. The event numbers are not identical, but I think that's probably mostly due to slightly different grouping resulting in different start dates. Not an issue as we use the numbers mostly to decide for the priority and they're close enough.

**Comment by dbumblis-parabol** [comment]
> Idea: during this work, make it optional to use the APM. In many environments there is no APM and it shouldn't be mandatory to use. Example: self-hosted, PubSec envs, PPMIs...
> 
> If enabled, send stuff to the APM, if disabled, just throw logs!

+1
